Sources/PrimerSDK/Classes/Services/API/Primer/PrimerAPI.swift

```diff
@@ -280,6 +279,8 @@ internal extension PrimerAPI {
                 return try? JSONEncoder().encode(request)
             } else if let request = paymentMethodTokenizationRequest as? BankSelectorTokenizationRequest {
                 return try? JSONEncoder().encode(request)
+            } else if let request = paymentMethodTokenizationRequest as? BlikPaymentMethodTokenizationRequest {
+                return try? JSONEncoder().encode(request)
```

Comment:
As I can see from the lines above we are starting to add multiple checks in order to receive the correct encoded object - we can think about whether we may think of other ways together, later on, 💪 but for now is 👌 🆗